### PR TITLE
Stat: Percent change refinements

### LINKED ISSUE #79716: Stat: Percent Change Styling Refinement
For Stat Panel Percent Change, during horizontal layout, align value to the right (next to percent change):
Image

Also, during vertical orientation, value size needs to be more subdued:
Image

Related to https://github.com/grafana/grafana/pull/78250

For additional context see [this slack message](https://raintank-corp.slack.com/archives/C04J73AAQ87/p1702912165233159)

### PR DESCRIPTION
**Pull request by nmarrs**
Initial approach at addressing some follow-up feedback for stat percent change

1. Align value to right (next to percent change) when horizontal
Not super happy with current approach taken here with modifying html layout / potential offset of values / percent as each % / value pairing is in a separate flex layout / not combined - given this the argument for keeping value centered could be made 🤷‍♂️ 
Screenshot 2024-01-03 at 10 15 08 PM

2. Reduce value size in vertical orientation when percent change is visible 

Fixes #79716

### CODE REVIEW

**Comment by drew08t** [comment]
Added a commit for item 2, reducing value font to 80% when stacked and percent change is present.

Without percent change:
Jan-10-2024 23-04-51

With percent change:
Jan-10-2024 23-04-26

**Review by drew08t — COMMENTED** [review]
I think for number 1, if we could align "columns" for the entire panel, it may be easier to read.

File: packages/grafana-ui/src/components/BigValue/BigValue.tsx [hunk 1]
```diff
@@ -99,15 +100,36 @@ export class BigValue extends PureComponent<Props> {
...
+              value={textValues}
+              style={{ ...valueStyles, fontSize: Number(valueStyles.fontSize) * 0.8 }}
+            />
+            <PercentChange percentChange={percentChange} styles={layout.getPercentChangeStyles(percentChange)} />
+          </>
+        );
+      } else {
+        return <FormattedValueDisplay value={textValues} style={valueStyles} />;
+      }
+    };
```
Comment (nmarrs) [inline packages/grafana-ui/src/components/BigValue/BigValue.tsx:125]: not happy with this approach but not sure how else to get value / % change to be grouped together / flex styles applied

**Comment by Develer** [comment]
Just an idea, maybe we should keep the value always stick to the right side (in horizontal mode). The value seems more important than percent. Also, percent looks like optional to show and disable by default. It should add more consistency in design and probably will solve our aligning issue 🙂

I've played a bit with [figma](https://www.figma.com/file/pHB4HBHtsRYZRJQfYDBiaW/Stat?type=design&node-id=0%3A1&mode=design&t=ibvefXSEkEOAfIqg-1). Let me know what you think.

cc @lukasztyrala

**Comment by Develer** [comment]
After talking to @lukasztyrala and a couple of experiments, turned out the grid solution didn't work well as we can have long values or percent values which will affect the layout not in a good way 🙂

However, here is an alternative for aligning values. We can just put the percent value right under the main value. This solution will keep the main value on the right side, which is a good place for the main value (not in the middle between percent and label, doesn't make sense to me).

Here is the [PR](https://github.com/grafana/grafana/pull/80733) as an alternative with attached before and after videos inside.

**Pull request closed without merging**